libs/partners/anthropic/tests/integration_tests/test_chat_models.py

```diff
@@ -5,15 +5,15 @@
 
 
 def test_stream() -> None:
-    """Test streaming tokens from OpenAI."""
+    """Test streaming tokens from anthropic."""
```

Comment:
I think they go with uppercase A;

```suggestion
    """Test streaming tokens from Anthropic."""
```